src/pages/Events/EventInfo/EventInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, useEffect } from 'react';
+
+import { useParams } from 'react-router-dom';
+import { useTypedSelector } from '../../../hooks/useTypedSelector';
+
+import { useDispatch } from 'react-redux';
+import { getEventAction } from '../../../store/actions/eventsActions';
+
+import EventsList from '../EventsList/EventsList';
+import LoadingAnimation from '../../../components/LoadingAnimation/LoadingAnimation';
+import './EventInfo.scss';
+
+const EventInfo: FC = () => {
+  const dispatch = useDispatch();
+
+  const { id } = useParams();
+
+  const { loading, event } = useTypedSelector((state) => state.events);
```

Comment:
Usually useTypedSelector and useState should be in the beginning of the component